test_scripts/API/VehicleData/GetVehicleData/007_GetVD_app_version_is_greater_than_parameter_version.lua

```diff
@@ -0,0 +1,41 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
Script is executed successful but without steps at all, the same to 008_GetVD script

Reply:
@GetmanetsIrina I have another result. There are steps executed for parameters with version defined, e.g. `cloudAppVehicleID`, `electronicParkBrakeStatus` etc.
Have you used appropriate MOBILE/HMI API in ATF?